### PR TITLE
keyboard driver changes by glowman and other stuff.

### PR DESCRIPTION
**Pull request by JartC0ding**

### CODE REVIEW

File: disk_resources/resources/foxe_help.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Foxe Cheatsheet:
+
+To do anything else than just writing and deleting chars use <ESC> to go into edit mode.
+You should see that the mode in the statusbar has changed now. You can now use all the commands listed below!
+
+Moving around:
+    w - up a line 
+    s - down a line
+    a - left one char              #
```
Comment (ImDaBigBoss) [inline disk_resources/resources/foxe_help.txt:9]: Why is there a #?

File: Makefile [hunk 2]
```diff
@@ -44,7 +44,7 @@ vdi: img
 qcow2: img
 	qemu-img convert foxos.img -O qcow2 foxos.qcow2
 
-run: img ./tmp/ovmf
+run: ./tmp/ovmf
```
Comment (ImDaBigBoss) [inline Makefile:47]: @Glowman554 do you want to do this? I'm fine with that.

**Review by ImDaBigBoss — COMMENTED** [review]
It looks good. I just have 2 things to look out for.

File: Makefile (hunk 2 above)
Comment (Glowman554) [reply Makefile:47]: That's only that he could use the make command to run it on Mac we should keep it for everybody else

**Comment by JartC0ding** [comment]
Ye that one actually does nothing. I first needed to fork the other repos. Hence there is nothing useful here. So i will just close this request and leave the others open

File: Makefile (hunk 2 above)
Comment (ImDaBigBoss) [reply Makefile:47]: Well actually. Run is the same on all platforms. Maybe just make them 2 different commands.

**Pull request closed without merging**